src/app/clusters/scenes-server/SceneHandlerImpl.cpp

```diff
@@ -16,10 +16,156 @@
  */
 
 #include <app/clusters/scenes-server/SceneHandlerImpl.h>
+#include <app/util/endpoint-config-api.h>
 
 namespace chip {
 namespace scenes {
 
+namespace {
+
+using ConcreteAttributePath  = app::ConcreteAttributePath;
+using AttributeValuePairType = app::Clusters::ScenesManagement::Structs::AttributeValuePair::Type;
+
+/// ConvertByteArrayToUInt64
+/// @brief Helper function to convert a byte array to a uint64_t value
+/// @param EmberAfDefaultAttributeValue & defaultValue
+/// @param len Length of the byte array
+/// @return uint64_t or int64_t Value
+/// @note The attribute table supports 8 bytes only for unsigned integers, 4 bytes is the maximum for signed integers
```

Comment:
Really?  I am sure we have signed 64-bit things, in general....

Reply:
Either way, removed the note from the comment.